Documentation/Troubleshooting/troubleshooting.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Troubleshooting"
+---
+
+To help troubleshoot your Koor Storage Distribution clusters, here are some tips on what information will help solve the issues you might be seeing.
+If after trying the suggestions found on this page and the problem is not resolved, the Koor Storage Distribution team is very happy to help you troubleshoot the issues through [GitHub Discussions](https://github.com/koor-tech/koor/discussions).
+
+## Troubleshooting Techniques
+
+Kubernetes status and logs are the main resources needed to investigate issues in any Rook cluster.
+
+## Kubernetes Tools
+
+Kubernetes status is the first line of investigating when something goes wrong with the cluster. Here are a few artifacts that are helpful to gather:
+
+### Rook Pod Status
+
+```console
+kubectl get -n <cluster-namespace> pod -o wide`
+kubectl get -n rook-ceph pod -o wide
+```
+
+### Logs for Rook Pods
+
+Logs for the operator Pod:
+
+```console
+kubectl logs -n <cluster-namespace> -l app=<storage-backend-operator>
+kubectl logs -n rook-ceph -l app=rook-ceph-operator
+```
```

Comment:
Why some of these have examples and others not?

Reply:
Now all code blocks should have an example